bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -5484,9 +5488,11 @@ void ReplicaImp::sendResponses(PrePrepareMsg *ppMsg, IRequestsHandler::Execution
                                                                              req.outReply,
                                                                              req.outActualReplySize,
                                                                              req.outReplicaSpecificInfoSize);
+        replyMsg->setResult(req.outExecutionStatus);
```

Comment:
It's better to pass the execution result (here: req.outExecutionStatus) into allocateReplyFromSavedOne and to set it there.

Reply:
passed in allocateNewReplyMsgAndWriteToStorage and set from there